### PR TITLE
feature(spring-mvc): detect endpoint methods with either protected or package-private visibility

### PR DESCRIPTION
**Pull request by murdos**
This methods are valid Spring endpoints, so they should be correctly detected

### CODE REVIEW

**Comment by murdos** [comment]
Requires #168

File: openapi-maven-plugin/src/main/java/io/github/kbuntrock/reflection/ReflectionsUtils.java [hunk 1]
```diff
@@ -39,20 +37,6 @@ public static ClassLoader getProjectClassLoader() {
 		return ReflectionsUtils.projectClassLoader;
 	}
 
-	public static ConfigurationBuilder createConfigurationBuilder() {
```
Comment (kbuntrock) [inline openapi-maven-plugin/src/main/java/io/github/kbuntrock/reflection/ReflectionsUtils.java:42]: I like this! the "testMode" boolean is not needed anymore now, I will remove it.

File: openapi-maven-plugin/src/test/java/io/github/kbuntrock/SpringClassAnalyserTest.java [hunk 2]
```diff
@@ -698,6 +701,14 @@ public void collection() throws MojoFailureException, IOException, MojoExecution
 		checkGenerationResult("ut/SpringClassAnalyserTest/collection.yml", generated.get(0));
 	}
 
+	@Test
+	public void package_private() throws MojoFailureException, IOException, MojoExecutionException {
```
Comment (kbuntrock) [inline openapi-maven-plugin/src/test/java/io/github/kbuntrock/SpringClassAnalyserTest.java:705]: perfect, tests are a must have! 👍

**Review by kbuntrock — APPROVED** [review]
Thank you for this nice PR, nothing much to say

**Pull request merged**